safe/gui/tools/peta_bencana_dialog.py

```diff
@@ -86,14 +92,36 @@ def __init__(self, parent=None, iface=None):
         self.parent = parent
         self.setupUi(self)
 
-        self.setWindowTitle(self.tr('PetaJakarta Downloader'))
+        title = self.tr('PetaBencana Downloader')
+        self.setWindowTitle(title)
 
         self.iface = iface
 
+        self.source = None
+
+        self.radio_button_group = QButtonGroup()
+        self.radio_button_group.addButton(self.radio_button_production)
+        self.radio_button_group.addButton(self.radio_button_development)
+
+        self.radio_button_group.setExclusive(True)
+        self.radio_button_production.setChecked(True)
+        self.populate_combo_box()
+
+        settings = QSettings()
+        developer_mode = settings.value('inasafe/developer_mode', False,
```

Comment:
![selection_021](https://cloud.githubusercontent.com/assets/11134669/23030371/29dc7cc8-f4a0-11e6-9de4-323931960146.png)
but mine is unicode :(